pkg/controller/container-runtime-config/helpers_test.go

```diff
@@ -1245,22 +1262,227 @@ func TestGetValidScopePolicies(t *testing.T) {
 }
 
 func TestGenerateSigstoreRegistriesConfig(t *testing.T) {
-	testClusterImagePolicyCR0 := clusterImagePolicyTestCRs()["test-cr0"]
-	testClusterImagePolicyCR1 := clusterImagePolicyTestCRs()["test-cr1"]
 
-	expectSigstoreRegistriesConfig := []byte(
-		`docker:
-  test0.com:
+	// testcase CIP scopes:
+	// "a.com/a1/a2",
+	// "a.com/a1/a2@sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+	// "*.example.com",
+	// "policy.scope"
+	testClusterImagePolicyCR2 := clusterImagePolicyTestCRs()["test-cr2"]
+
+	clusterScopePolicies, err := getValidScopePolicies([]*apicfgv1alpha1.ClusterImagePolicy{&testClusterImagePolicyCR2})
+	require.NoError(t, err)
+
+	type testcase struct {
+		name                             string
+		clusterScopePolices              map[string]signature.PolicyRequirements
```

Comment:
```suggestion
		clusterScopePolicies              map[string]signature.PolicyRequirements
```
but the field is always set to the same value, so it can also be removed instead.